helios-services/src/main/java/com/spotify/helios/master/ZooKeeperMasterModel.java

```diff
@@ -123,22 +102,9 @@ public ZooKeeperMasterModel(final ZooKeeperClientProvider provider) {
    */
   @Override
   public void registerHost(final String host, final String id) {
-    log.info("registering host: {}", host);
```

Comment:
Basically, I moved `registerHost()` and `deregisterHost()` and any helper methods to a new class `ZooKeeperRegistrarUtil` since I needed to reuse that logic in `AgentZooKeeperRegistrar`.